app/(app)/offence/result.tsx

```diff
@@ -10,18 +10,17 @@ import ScreenViewCentered from '@/components/screen-layout/ScreenViewCentered'
 import IconButton from '@/components/shared/IconButton'
 import { useClearHistory } from '@/hooks/useClearHistory'
 import { getRoleByKey } from '@/modules/backend/constants/roles'
-import { ScanResultEnum } from '@/modules/backend/openapi-generated'
 import { getDefaultOffenceStateByRole } from '@/state/OffenceStore/getDefaultOffenceStateByRole'
 import { useOffenceStoreContext } from '@/state/OffenceStore/useOffenceStoreContext'
 import { useSetOffenceState } from '@/state/OffenceStore/useSetOffenceState'
 
 type OffenceResultSearchParams = {
-  scanResult: ScanResultEnum
+  offenceResult: 'success' | 'error'
```

Comment:
generally i'm a fan of using generated types that are shared via imports over ad-hoc type enums but i havent' looked into this too closely